CHANGELOG.md

```diff
@@ -1,4 +1,52 @@
-### Changes in 0.19
+### Changes in 0.20
+
+- Fixed: error on some generated pages from missing sourcemap [#1019](https://github.com/luckyframework/lucky/pull/1019)
+- Updated: `options_for_select` to be a little more generic [#295](https://github.com/luckyframework/lucky/pull/295)
+- Added: ability to pass boolean attrs in link helper methods [#1032](https://github.com/luckyframework/lucky/pull/1032)
+- Removed: ability to set predicate methods on `needs` [#1034](https://github.com/luckyframework/lucky/pull/1034)
```

Comment:
I'd say maybe this is "Updated/removed" :P You now no longer need to use `needs` with a predicate var because Lucky will generate a `?` method for you if the type is a Bool.

Not sure how to put that, but I'd say this is nicer since you don't need to do `Component.new(signed_in?: true)` which is super weird